packages/rsocket-core/src/RSocketMachine.js

```diff
@@ -784,13 +784,14 @@ class RSocketMachineImpl<D, M> implements RSocketMachine<D, M> {
   }
 
   _handleRequestChannel(streamId: number, frame: RequestChannelFrame): void {
+    const payload = this._deserializePayload(frame);
```

Comment:
In all `_handleXXXX` methods, the frame is deserialized first. So it should be deserialized here as well.